<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prepare</class>
 <widget class="QWidget" name="Prepare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="totalLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="buttonLoad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textInformation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupRemoveNonvalidValue">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>Remove non-valid value</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioRemoveNonvalidCases">
              <property name="text">
               <string>Remove invalid Cases</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioRemoveNonvalidFeatures">
              <property name="text">
               <string>Remove invalid Features</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonRemove">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupSeparate">
         <property name="title">
          <string>Separate Train and Test</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Percentage of Testing data</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loadTestingReference">
              <property name="text">
               <string>Load Test Dataset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearTestingReference">
              <property name="text">
               <string>Clear Test Dataset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinBoxSeparate">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.300000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkSeparate">
              <property name="text">
               <string>Separate Train And Test</string>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Balance Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioNoneBalance">
              <property name="text">
               <string>None Balance</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioUpSampling">
              <property name="text">
               <string>Upsampling</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioDownSampling">
              <property name="text">
               <string>Downsampling</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioSmote">
              <property name="text">
               <string>SMOTE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Check and Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableFeature"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
